### PR TITLE
Fix Fade Animation

### PR DESCRIPTION
**Pull request by Anutha02**
The current implementation of the fade animation is flawed. This commit fixes it.

### CODE REVIEW

**Comment by Anutha02** [comment]
@paolorotolo please review

**Comment by Anutha02** [comment]
@cortinico Please review

File: appintro/src/main/java/com/github/paolorotolo/appintro/internal/viewpager/ViewPagerTransformer.kt [hunk 1]
```diff
@@ -56,13 +56,16 @@ internal class ViewPagerTransformer(
             }
             TransformType.FADE -> {
                 if (position <= -1.0f || position >= 1.0f) {
+                    page.translationX = page.width*position
```
Comment (cortinico) [inline appintro/src/main/java/com/github/paolorotolo/appintro/internal/viewpager/ViewPagerTransformer.kt]: Please change with:
```suggestion
                    page.translationX = page.width.toFloat()
```

File: appintro/src/main/java/com/github/paolorotolo/appintro/internal/viewpager/ViewPagerTransformer.kt [hunk 2]
```diff
@@ -56,13 +56,16 @@ internal class ViewPagerTransformer(
...
                     page.alpha = 0.0f
                     page.isClickable = false
                 } else if (position == 0.0f) {
+                    page.translationX = page.width*position
```
Comment (cortinico) [inline appintro/src/main/java/com/github/paolorotolo/appintro/internal/viewpager/ViewPagerTransformer.kt]: Please change with:
```suggestion
                    page.translationX = 0.0f
```

**Comment by Anutha02** [comment]
@cortinico I will check and apply suggestion soon

**Pull request merged**